express-api/src/routes/buildingsRouter.ts

```diff
@@ -22,11 +21,8 @@ router
   .get(catchErrors(getBuilding))
   .put(catchErrors(updateBuilding))
   .delete(catchErrors(deleteBuilding));
-router
-  .route(BUILDINGS_ROUTE)
-  .get(catchErrors(filterBuildingsQueryString))
-  .post(catchErrors(addBuilding));
-router.route(`${BUILDINGS_ROUTE}/filter`).post(catchErrors(filterBuildingsRequestBody));
+router.route(BUILDINGS_ROUTE).get(catchErrors(getBuildings)).post(catchErrors(addBuilding));
+router.route(`${BUILDINGS_ROUTE}/filter`).post(catchErrors(getBuildings));
```

Comment:
We can get rid of the filter route. That kind of get request is covered in the above route now.

Also, check out the parcelsRouter for an example of this: Because the `/buildings` part is now assumed from express.ts, we can drop the BUILDINGS_ROUTE prefix on all these routes.